models/articles.models.js

```diff
@@ -15,5 +15,16 @@ exports.fetchArticles = () => {
       });
       return deleteUnuseItem;
     });
- 
+};
+
+exports.fetchArticlesById = (id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = ${id};`)
```

Comment:
use `$` syntax with a values array to prevent SQL injection

Reply:
I made the changes Thanks for your comments!